Error: Cannot find module 'underscore.string'

```sh
yo gas-mithril
```

An error happened.
```
Error: Cannot find module 'underscore.string'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
```

--- a/package.json
+++ b/package.json
@@ -16,8 +16,9 @@
     "yeoman-generator"
   ],
   "dependencies": {
-    "yeoman-generator": "^1.0.1",
     "chalk": "^1.1.3",
+    "underscore.string": "^3.3.4",
+    "yeoman-generator": "^1.0.1",
     "yosay": "^2.0.0"
   },
   "devDependencies": {
@@ -32,7 +33,6 @@
     "gulp-mocha": "^3.0.1",
     "gulp-nsp": "^2.4.1",
     "gulp-plumber": "^1.0.0",
-    "underscore.string": "^3.2.2",
     "yeoman-assert": "^3.0.0"
   },
   "repository": "fossamagna/generator-gas-mithril",

